git.md

```diff
@@ -211,17 +231,17 @@ The relevant commits are all rewritten to be brought on [top of the `HEAD` of th
 
 ## On Reviewing
 
-Try to set aside some quality time as a routine to proactively review others' pull/merge requests, as thoroughly as your bandwidth allows. If there's no time, and/or no willingness, do not just cast approval without reviewing.
+Try to set aside some quality time as a routine to proactively review others' pull/merge requests, as thoroughly as your bandwidth allows. If there's no time, and/or no willingness, do not just cast approve without reviewing. For GitHub repositories, this is the perfect time to make good use of [GitHub's notifcation with a `review-requested` filter](https://github.com/notifications?query=reason%3Areview-requested).
 
 Actively self-review and monitor your own pull/merge requests -- be on the lookout for potential conflicts after other requests have been merged.
 
 When giving code change requests, think twice before you submit -- be reasonable and constructive, and be ready to defend your reasons.
 
 If the code change suggestion does not warrant definitive reasoning, but would still be constructive in your opinion, make it as a non-blocking comment.
 
-Where relevant, go through the changes in deploy preview or in your local environment to check for UI/UX flow as well as bugs.
+Where applicable, go through the changes in deploy preview or in your local environment to go through intended workflow, and check for potential issues.
 
-Prioritize on reviewing those that marked as bug fixes, then the new features. Use `G2M` as a search filter to minimize noise -- provided that it is a well-followed convention in the given project.
+Prioritize on reviewing those that marked as bug fixes, then the new features. Use `G2M` as a search filter to minimize noise -- provided that it is a well-followed convention in the given repository.
```

Comment:
suggestion: should be 'reviewing those that are marked as bug fixes'

also what does 'it' refer to in the line 'provided that it is a well-followed convention in the given repository'? PRs?

Reply:
it - PR prefix. will include in the revision